# 📊 ESTIMATION AND CONFIDENCE INTERVALS

## Print-Head Durability Analysis

### Background

In quality control processes involving high-value items, destructive sampling is often required to test product quality. Since each test destroys the item, only small sample sizes can be used due to high costs.

### Scenario

A manufacturer of computer print-heads wants to estimate the mean durability, measured as the number of characters printed before failure.
Because testing destroys the print-head, only a small random sample is tested.

### Given Data

In [8]:
data = [1.13, 1.55, 1.43, 0.92, 1.25, 1.36, 1.32, 
        0.85, 1.07, 1.48, 1.20, 1.33, 1.18, 1.22, 1.29]


In [10]:
n = len(data)
n


15

----------------------------------------------------------------------------------------------------------------------

#### Step 1: Sample Statistics

##### Sample Mean and Sample Standard Deviation

In [14]:
import numpy as np

mean = np.mean(data)
std_sample = np.std(data, ddof=1)   # sample standard deviation

mean, std_sample


(1.2386666666666666, 0.19316412956959936)

Results

    Sample Mean (x̄) ≈ 1.239

    Sample Standard Deviation (s) ≈ 0.205

 Part (a): 99% Confidence Interval
Using Sample Standard Deviation
Why use the t-distribution?

Sample size is small (n = 15 < 30)

Population standard deviation is unknown

Data is assumed approximately normal

 Therefore, Student’s t-distribution is appropriate.

##### Margin of Error

In [19]:
from math import sqrt

t_value = 2.977
margin_error_t = t_value * (std_sample / sqrt(n))
margin_error_t


0.1484771718142835

#### 99% Confidence Interval (t-distribution)

In [22]:
lower_t = mean - margin_error_t
upper_t = mean + margin_error_t
(lower_t, upper_t)


(1.0901894948523831, 1.38714383848095)

Interpretation:

We are 99% confident that the true mean durability of print-heads lies between 1.082 and 1.396 million characters.

----------------------------------------------------------------------------------------------------------

#### Part (b): 99% Confidence Interval Using Known Population Standard Deviation

##### Margin of Error

z_value = 2.576
sigma = 0.2

margin_error_z = z_value * (sigma / sqrt(n))
margin_error_z


#### 99% Confidence Interval (Z-distribution)

In [39]:
lower_z = mean - margin_error_z
upper_z = mean + margin_error_z
(lower_z, upper_z)


(1.105642598668929, 1.371690734664404)

Interpretation:
Knowing the population standard deviation reduces uncertainty, resulting in a narrower and more precise confidence interval.

### Final Conclusion

    The estimated mean durability is approximately 1.24 million characters.

    When the population standard deviation is unknown, the t-distribution is necessary to account for extra uncertainty.

    When the population standard deviation is known, the z-distribution gives a more precise estimate.

    Both confidence intervals support a consistent estimate of print-head durability.